pkg/operator/workloads/api_workload.go

```diff
@@ -22,14 +22,13 @@ import (
 	kapps "k8s.io/api/apps/v1"
```

Comment:
Since there are lots of other files (e.g. 3 others that start with `api*`), I thought it might be nice to make it clear that this the workload implementation for apis